[API Shield] FAQ to docs (#25656)
cloudflare/cloudflare-docs: src/content/docs/api-shield/security/volumetric-abuse-detection.mdx

Cloudflare Volumetric Abuse Detection helps you set up a system of adaptive rate limiting.

> After [API Discovery](/api-shield/security/api-discovery/), Cloudflare looks for <GlossaryTooltip term="API endpoint">endpoint</GlossaryTooltip> abuse based on common user traffic.

Author: ```suggestion
Cloudflare looks for endpoint abuse based on user traffic to individual endpoints.
```

## Process

Volumetric Abuse Detection analyzes your API's individual session traffic statistics to recommend per-endpoint, per-session rate limits.

To access your endpoints: 

Old dashboard: **Security** > **API Shield** > **Endpoint Management**

New dashboard: **Security** > **Web Assets** > **Endpoints**

Recommendations will continue to update if your traffic pattern changes.

### Requirements

Volumetric Abuse Detection generates rate limit thresholds only after collecting sufficient, statistically safe traffic data for an endpoint. If recommendations are missing for a discovered endpoint, the traffic likely failed to meet the necessary criteria.

Thresholds are suggested only for endpoints that satisfy all of the following requirements within the last seven days (or since initial discovery):

- The endpoint must receive sufficient valid traffic (traffic that meets the [API Discovery](/api-shield/security/api-discovery/#requirements) criteria). Intermittent or erratic traffic may prevent suggestions.

> - The endpoint must be accessed by at least 50 distinct sessions in any 24-hour period during the last 7 days.

Reviewer: ```suggestion
- The endpoint must be accessed by at least 50 distinct sessions in any 24-hour period during the last seven days.
```

## Process

Volumetric Abuse Detection analyzes your API's individual session traffic statistics to recommend per-endpoint, per-session rate limits.

To access your endpoints: 

Old dashboard: **Security** > **API Shield** > **Endpoint Management**

New dashboard: **Security** > **Web Assets** > **Endpoints**

Recommendations will continue to update if your traffic pattern changes.

### Requirements

Volumetric Abuse Detection generates rate limit thresholds only after collecting sufficient, statistically safe traffic data for an endpoint. If recommendations are missing for a discovered endpoint, the traffic likely failed to meet the necessary criteria.

Thresholds are suggested only for endpoints that satisfy all of the following requirements within the last seven days (or since initial discovery):

> - The endpoint must receive sufficient valid traffic (traffic that meets the [API Discovery](/api-shield/security/api-discovery/) criteria). Intermittent or erratic traffic may prevent suggestions.

Author: ```suggestion
- The endpoint must receive sufficient valid traffic (traffic that meets the [API Discovery](/api-shield/security/api-discovery/#requirements) criteria). Intermittent or erratic traffic may prevent suggestions.
```

> Volumetric Abuse Detection analyzes your API’s individual session traffic statistics to recommend per-endpoint, per-session rate limits.

Author: ```suggestion
Volumetric Abuse Detection analyzes your API's individual session traffic statistics to recommend per-endpoint, per-session rate limits.
```
